src/gtpV2Codec/ieClasses/arpIe.cpp

```diff
@@ -68,6 +74,13 @@ bool ArpIe::decodeArpIe(MsgBuffer &buffer, ArpIeData &data, Uint16 length)
         errorStream.add("Attempt to read beyond IE boundary: pl\n");
         return false;
     }
+    buffer.skipBits(1);
```

Comment:
In Nucleus this is autogenerated code. @hariharanb77  Can we get this fix in autogeneration ?

Reply:
Hi Ajay,
Will check this.
thx.